src/vorta/views/repo_tab.py

```diff
@@ -129,7 +129,7 @@ def ssh_copy_to_clipboard_action(self):
 
     def compression_select_action(self, index):
         profile = self.profile()
-        profile.compression = self.repoCompression.currentData()
+        profile.compression = self.repoCompression.currentData() or ''
```

Comment:
this seems to be unrelated to the topic of your PR.

Reply:
Oops, you're right. I had to add that because after applying migrations to the settings database the program ended with an error on that line because of the NOT NULL constraint on that field.

Should I remove that file from the PR?

Reply:
We recently changed the available compression options. They should be migrated when you run a version later than commit 3efecd71f3b0fc69261284efcd8d628bb2b89d59.